.gitignore

```diff
@@ -46,3 +46,17 @@ sysinfo.txt
 # Builds
 *.apk
 *.unitypackage
+Moonscraper Chart Editor/Custom Resources/drun textures/5_sp_kick_2x.png
+Moonscraper Chart Editor/Custom Resources/drun textures/5_sp_2xkick.png
+Moonscraper Chart Editor/Custom Resources/drun textures/5_reg_kick_2x.png
+Moonscraper Chart Editor/Custom Resources/drun textures/5_reg_2xkick.png
+Moonscraper Chart Editor/Custom Resources/drun textures/4_sp_cymbal.png
+Moonscraper Chart Editor/Custom Resources/drun textures/4_reg_cymbal.png
+Moonscraper Chart Editor/Custom Resources/drun textures/3_sp_cymbal.png
+Moonscraper Chart Editor/Custom Resources/drun textures/3_reg_cymbal.png
+Moonscraper Chart Editor/Custom Resources/drun textures/2_sp_cymbal.png
+Moonscraper Chart Editor/Custom Resources/drun textures/2_reg_cymbal.png
+Moonscraper Chart Editor/Custom Resources/drun textures/1_sp_cymbal.png
+Moonscraper Chart Editor/Custom Resources/drun textures/1_reg_cymbal.png
+Moonscraper Chart Editor/Custom Resources/drun textures/0_sp_cymbal.png
+Moonscraper Chart Editor/Custom Resources/drun textures/0_reg_cymbal.png
```

Comment:
Don't put this in the gitignore. You should not be adding personal files to the ignore list like this. If you want to exclude personal files, never commit them in the first place. 
That being said, it would actually be better to commit these to the repro so that we always have drum assets to test with. If you are concerned with copyright, just replace these with dummy sprites for the drums instead.

Reply:
Also "drun textures"

Reply:
'drun' is funny, didn't notice that until it got committed lol